android/modules/media/src/java/ti/modules/titanium/media/TiCameraActivity.java

```diff
@@ -641,10 +641,14 @@ private void openCamera(int cameraId)
 			camera = null;
 		}
 
-		if (cameraId == Integer.MIN_VALUE) {
-			camera = Camera.open();
-		} else {
-			camera = Camera.open(cameraId);
+		try {
+			if (cameraId == Integer.MIN_VALUE) {
+				camera = Camera.open();
+			} else {
+				camera = Camera.open(cameraId);
+			}
+		} catch (Exception e) {
```

Comment:
And would it be better to use `RuntimeException` instead of a general `Exception`?